docs/keybindings/Keybindings_en.md

```diff
@@ -268,7 +268,6 @@ _This file is auto-generated. To update, make the changes in the pkg/i18n direct
   <kbd>►</kbd>: select next hunk
   <kbd>ctrl+o</kbd>: copy the selected text to the clipboard
   <kbd>e</kbd>: edit file
-  <kbd>o</kbd>: open file
```

Comment:
this keybinding was erroneously duplicated with the previous cheatsheet generation code